StockTicker/app/src/main/java/com/example/stockticker/view/MainActivity.kt

```diff
@@ -1,20 +1,23 @@
 package com.example.stockticker.view
 
-import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
 import androidx.lifecycle.Observer
 import com.example.stockticker.MainActivityViewModel
 import com.example.stockticker.R
+import dagger.hilt.android.AndroidEntryPoint
 import kotlinx.android.synthetic.main.activity_main.*
 import kotlinx.coroutines.ExperimentalCoroutinesApi
 import kotlinx.coroutines.FlowPreview
-import org.koin.android.viewmodel.ext.android.viewModel
+import javax.inject.Inject
 
 @ExperimentalCoroutinesApi
 @FlowPreview
+@AndroidEntryPoint
 class MainActivity : AppCompatActivity() {
 
-    private val viewModel: MainActivityViewModel by viewModel()
+    @Inject
+    lateinit var viewModel: MainActivityViewModel
```

Comment:
If this were a fragment and the view model used the `@ViewModelInject` annotation instead, you could instead just do:

`private val viewModel: MainActivityViewModel by viewModels()`